launch/turtlebot_sim_bringup_launch.xml

```diff
@@ -0,0 +1,16 @@
+<launch>
+  <node pkg="rviz2" exec="rviz2" name="rviz" args="--display-config $(find-pkg-share nav2_bringup)/rviz/nav2_default_view.rviz"/>
```

Comment:
I'm assuming that this rviz config does not include the route server markers. We should probably point to the one that will be in c1t_bringup following this PR: https://github.com/usdot-fhwa-stol/c1t_bringup/pull/16